zcash_client_backend/src/data_api.rs

```diff
@@ -139,6 +139,21 @@ pub enum NullifierQuery {
     All,
 }
 
+/// An intent of representing spendable value to reach a certain targeted
+/// amount.  `MinValue(Zatoshis)` refers to the amount of `Zatoshis` that can cover
+/// at minimum the given zatoshis that is conformed by the sum of spendable notes.
+///
+/// Discussion: why not just use ``Zatoshis``?
+///
+/// the `Zatoshis` value isn't enough to explain intent when seeking to match a
+/// given a given amount. Is the value expressed in `Zatoshis` the ceiling value
+/// or the minimum value of a given spend intent? How would you express that the
+/// value spend intent is "as much as possible" without knowing the value upfront?
+#[derive(Debug, Clone, Copy)]
+pub enum TargetValue {
+    MinValue(Zatoshis),
```

Comment:
```suggestion
    AtLeast(Zatoshis),
```
nonblocking, but I think this would be a better name.